components/automate-ui/src/app/entities/users/user.effects.ts

```diff
@@ -43,11 +44,10 @@ export class UserEffects {
     private store$: Store<NgrxStateAtom>
   ) { }
 
-  // 7/6/18: TODO in follow-up PR: fetch teams for all users - bd
   @Effect()
   getUsers$ = combineLatest([
     this.actions$.pipe(ofType<GetUsers>(UserActionTypes.GET_ALL)),
-    this.store$.select(iamMajorVersion)])
+    this.store$.select(iamMajorVersion).pipe(filter(identity))])
```

Comment:
Is the `filter(identity)` there to test that the version is not empty?

Reply:
yep! i added this because i noticed we sometimes called `getUsers` twice because it was letting through a nil `iamMajorVersion`, which was then set to a default value.